### PR TITLE
Issue 6266: Update EclipseLink from 2.7.3 to 2.7.4 for JPA 2.2

### LINKED ISSUE #6266: Update EclipseLink 2.7.3 to 2.7.4 for JPA 2.2
EclipseLink 2.7.4 has been released [here](http://www.eclipse.org/downloads/download.php?file=/rt/eclipselink/releases/2.7.4/eclipselink-plugins-2.7.4.v20190115-ad5b7c6b2a.zip)

### PR DESCRIPTION
**Pull request by dazey3**
fixes #6266

Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

File: dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/ToStringLocalizationResource.nlsprops [hunk 1]
```diff
@@ -127,43 +127,43 @@ server_name=server name
...
+staticweave_commandline_help_message_6of19=           Explicitly identify where META-INF/persistence.xml is stored.  It must be the root of META-INF/persistence.xml.
 
-staticweave_commandline_help_message_7of19=This option is typically used to specify the main jar when weaving files referenced by <jar-file> in persistence.xml.
+staticweave_commandline_help_message_7of19=           This option is typically used to specify the main jar when weaving files referenced by <jar-file> in persistence.xml.
 
-staticweave_commandline_help_message_8of19=-persistencexml
+staticweave_commandline_help_message_8of19=    -persistencexml
 
-staticweave_commandline_help_message_9of19=Identify the location of the persistence.xml relative to the root of the persistence unit if it is somewhere other that META-INF/persistence.xml
+staticweave_commandline_help_message_9of19=           Identify the location of the persistence.xml relative to the root of the persistence unit if it is somewhere other that META-INF/persistence.xml
```
Comment (jgrassel) [inline dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/ToStringLocalizationResource.nlsprops]: What's with all the spaces in the message text files?

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by dazey3** [comment]
#build

**Comment by Azquelt** [comment]
This has broken the build for external users
```
error  : Cannot find /error/org.eclipse.persistence.core;version=2.7.4.v20190115-ad5b7c6b2a Not found in [bnd-cache [/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/cache/4.1.0/bnd-cache r/w=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Local, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Release, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublic, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.ibmdhe/repository, storage=RemoteIBM, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=DummyRepo, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublish, inited=true, redeploy=false]]
error  : Cannot find /error/org.eclipse.persistence.jpa;version=2.7.4.v20190115-ad5b7c6b2a Not found in [bnd-cache [/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/cache/4.1.0/bnd-cache r/w=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Local, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Release, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublic, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.ibmdhe/repository, storage=RemoteIBM, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=DummyRepo, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublish, inited=true, redeploy=false]]
error  : org.eclipse.persistence.core;version=2.7.4.v20190115-ad5b7c6b2a Not found in [bnd-cache [/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/cache/4.1.0/bnd-cache r/w=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Local, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Release, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublic, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.ibmdhe/repository, storage=RemoteIBM, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=DummyRepo, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublish, inited=true, redeploy=false]]
error  : org.eclipse.persistence.jpa;version=2.7.4.v20190115-ad5b7c6b2a Not found in [bnd-cache [/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/cache/4.1.0/bnd-cache r/w=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Local, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/_8SuEYEJiEemZ2MYBJzNhRQ-EBC.PROD.WASRTC-6CX-001-00-00/jbe/build/open-liberty/dev/cnf/local, storage=Release, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublic, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.ibmdhe/repository, storage=RemoteIBM, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=DummyRepo, inited=true, redeploy=false], MavenBndRepository [localRepo=/home/jazz_build/.m2/repository, storage=RemotePublish, inited=true, redeploy=false]]
```

**Comment by tevans78** [comment]
Looks like version 2.7.4 of those bundles is in Maven Central but those very specific versions are not. If they were customized then the new versions should have been uploaded to DHE

**Pull request merged**